src/stories/Library/nav-grid/NavGrid.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC, ReactNode, useEffect } from "react";
+import clsx from "clsx";
+
+type CardProps = {
+  title?: string;
+  showSubtitles?: boolean;
```

Comment:
I dont understand this comment - right now, it is an optional value (e.g. `?`)

Reply:
I'm asking wether it should be possible to decide on the individual item, whether it should include subtitle or not, rather than control this on the parent. 

We'll just take it at demo. 

I guess this can be accomplished regardless right now, by allowing subtitles and just not filling a description for the individual item. 

So it does work both ways already in that sense.